packages/eslint-plugin-fbtee/src/__tests__/no-empty-strings-test.ts

```diff
@@ -0,0 +1,197 @@
+/* eslint-disable import/no-unresolved */
```

Comment:
Not sure why the import plugin doesn't like `@typescript-eslint/rule-tester`

Reply:
I'll take a look at that tomorrow (Friday).